src/caselawclient/models/documents.py

```diff
@@ -81,14 +81,14 @@ def public_uri(self) -> str:
     @cached_property
     def name(self) -> str:
         return self._get_xpath_match_string(
-            "//akn:FRBRname/@value",
+            "/akn:akomaNtoso/akn:*/akn:meta/akn:FRBRname/@value",
```

Comment:
Is there a reason for adding the `/akn:akomaNtoso/...` prefix?

Does that fix a current judgment specific bug, or just preempting any issues we might have with both judgment and press summary?

Reply:
Specificity; there can be other FRBRnames in annexes, and we'd like to cut down the number of things we have multiple names for.